_beats/vendor/vendor.json

```diff
@@ -521,7 +511,7 @@
 		},
 		{
 			"path": "github.com/elastic/go-structform/internal/visitors",
-			"revision": "",
+			"revision": "v0.0.4",
```

Comment:
This looks strange, also I could not find `github.com/elastic/go-structform/internal/visitors`, but `github.com/elastic/go-structform/visitors`. Same for `github.com/elastic/go-structform/internal/ubjson` above. 

That looks like a potential issue in beats though, https://github.com/elastic/beats/pull/7676